### ISSUE TITLE
Google storage file import -- KeyError: 'resource'

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#442]
Importing file from gs://storage-public-data/WorkflowInputFiles/b37/human_g1k_v37_decoy.fasta.pac...
   copying to destination gs://cgs-test-sequencing-results/cgs-test-sequencing-results/imported/2017-08-11T23.23.31Z_632f49e4_human_g1k_v37_decoy.fasta.pac ...
Traceback (most recent call last):
  File "/bin/loom", line 9, in <module>
    load_entry_point('loomengine==0.5.0', 'console_scripts', 'loom')()
  File "/opt/loom/loomengine/client/main.py", line 102, in main
    return Main().run()
  File "/opt/loom/loomengine/client/main.py", line 98, in run
    return self.args.SubcommandClass(self.args).run()
  File "/opt/loom/loomengine/client/file_client.py", line 274, in run
    self.args.SubSubcommandClass(self.args).run()
  File "/opt/loom/loomengine/client/file_client.py", line 67, in run
    retry=self.args.retry,
  File "/opt/loom/loomengine/utils/filemanager.py", line 522, in import_from_patterns
    force_duplicates=force_duplicates, retry=retry))
  File "/opt/loom/loomengine/utils/filemanager.py", line 534, in import_from_pattern
    retry=retry,
  File "/opt/loom/loomengine/utils/filemanager.py", line 549, in import_file
    return self._execute_file_import(data_object, source, retry=retry)
  File "/opt/loom/loomengine/utils/filemanager.py", line 721, in _execute_file_import
    source.copy_to(destination)
  File "/opt/loom/loomengine/utils/filemanager.py", line 147, in copy_to
    copier.copy()
  File "/opt/loom/loomengine/utils/filemanager.py", line 454, in copy
    self.source.blob, token=rewrite_token)
  File "/usr/lib/python2.7/site-packages/google/cloud/storage/blob.py", line 871, in rewrite
    self._set_properties(api_response['resource'])
KeyError: 'resource'

**Comment by nhammond** [comment]
The file is uploaded to google storage, but Loom file data object ends up in "upload_status": "failed" state.

This is seems sporadic. Of 45 file imports, 8 failed. On a later attempt, the number of failures was similar but on different files.

Using "loom file import --retry" does not retry for KeyError.

**Comment by nhammond** [comment]
Reproduced the failure iteractively:

```
>>> import google.cloud.storage.client
>>> client = google.cloud.storage.client.Client('cgstesting-0717')
>>> souce_bucket = client.get_bucket('storage-public-data')
>>> dest_bucket = client.get_bucket('cgs-test-sequencing-results')
>>> source_blob = souce_bucket.get_blob('WorkflowInputFiles/b37/gnomad.combined.r2.0.1.sites.allchr.ace.exome.cgs.vcf.gz')
>>> dest_blob = google.cloud.storage.blob.Blob('cgs-test-sequencing-results/imported/test', dest_bucket, chunk_size=1024*1024*100)
>>> rewrite_token = None
>>> rewrite_token, rewritten, size = dest_blob.rewrite(source_blob, token=rewrite_token)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/google/cloud/storage/blob.py", line 871, in rewrite
    self._set_properties(api_response['resource'])
KeyError: 'resource'
```

**Comment by nhammond** [comment]
Removing CHUNKS_SIZE had no effect.

After upgrading from google_cloud==0.23.0 to 0.26.0 the copy was successful.

**Comment by nhammond** [comment]
Fixed in 2b08be68a16ae43673e826a5fd3294d1c97d451c
